mappings/net/minecraft/item/MiningToolItem.mapping

```diff
@@ -19,3 +19,4 @@ CLASS avj net/minecraft/item/MiningToolItem
 		ARG 4 pos
 	METHOD a getBlockBreakingSpeed (Lawo;Lbpm;)F
 		ARG 1 stack
+		ARG 2 vBlockState3
```

Comment:
Is this intentional? It's a wee bit of a weird name, so wasn't sure if an artefact from Enigma or not. Either way, not a blocker.

Reply:
Enigma artifact...